lib/modules/apostrophe-login/index.js

```diff
@@ -428,40 +404,102 @@ module.exports = {
         const userSafe = await safe.findOne({
           _id: user._id
         });
+
         const perSince = getPerSince();
-        if (userSafe.attempts && userSafe.attempts.filter(attempt => (attempt >= perSince)).length >= allowedAttempts) {
-          await safe.update({
-            _id: user._id
-          }, {
-            $set: {
-              lockout: new Date()
-            }
-          });
-          return true;
+        const unallowedAttemps = userSafe.attempts && userSafe.attempts
```

Comment:
Typo: should be `unallowedAttempts` and `disallowedAttempts` would be better grammar. Actally let's just change it to `disallowed`, no need for the Attempts part of this variable name.

Reply:
Yes it's clearer!